tasks/main.yml

```diff
@@ -66,3 +66,35 @@
   changed_when: sonar_version | string not in api_check_version.content
   notify: fail if expected sonar version not confirmed
   ignore_errors: "{{ ansible_check_mode }}"
+
+- name: "Ensure old admin password match"
+  uri:
+    method: POST
+    url: "http://{{ sonar_web_host }}:{{ sonar_web_port }}{{ sonar_web_context }}/api/authentication/login"
+    force_basic_auth: true
+    url_username: "{{ sonar_admin_user }}"
+    url_password: "{{ sonar_admin_old_password }}"
+    body: "login={{ sonar_admin_user }}&password={{ sonar_admin_old_password }}"
```

Comment:
As a general rule of thumb, I would disable logging when credentials might leak to execution logs.